### PR TITLE
Fix Phantom Swapper duplicates error

### PR DESCRIPTION
**Pull request by Jam516**
@jeff-dude identified a duplicates error in the phantom swapper fee payments spell: https://github.com/duneanalytics/spellbook/pull/8417

Upon investigation, I found that the cause was a spambot called rocketpot which spams addresses with tiny amounts of SOL ([example](https://solscan.io/tx/3gmEqXALtN32LeDbm1LKPGAv83h8vAYjq5GvryJtBaNFmqNx4iBDNzR3vqTA5wps1EBHPX7sASEe67vQBtDodTf1))

To fix this, I join the solana trades table onto the fee payment query. This excludes edge cases like the rocketpot spamming, leaving only transactions where a phantom wallet is making a swap and a fee is being paid to phantom.

To test the fix, I made a query that contains the `phantom_swapper_fee_payments_raw` logic together with the new join and checks for recent duplicates. As you can see, no duplicates are found.
Fix - https://dune.com/queries/5370319
Original - https://dune.com/queries/5370136 (returns three duplicates found in that same time period)

### CODE REVIEW

**Comment by Jam516** [comment]
Note: Yes, the run times out, but this was also the case when the phantom swapper spell was first created and we got permission to merge it despite that

File: dbt_subprojects/solana/models/_sector/dex/phantom_swapper/phantom_swapper_solana_fee_payments_raw.sql [hunk 1]
```diff
@@ -29,6 +28,9 @@ with sol_payments as (
                 fee_addresses.fee_receiver = account_activity.address
                 and balance_change > 0
             )
+        join {{ source('dex_solana', 'trades') }} as trades
+            on trades.tx_id = account_activity.tx_id
+            and trades.block_time = account_activity.block_time
```
Comment (jeff-dude) [inline dbt_subprojects/solana/models/_sector/dex/phantom_swapper/phantom_swapper_solana_fee_payments_raw.sql:33]:
```suggestion
            and trades.block_time = account_activity.block_time
            {% if is_incremental() %} 
                {{ incremental_predicate('trades.block_time') }}
            {% else %} 
                trades.block_time >= timestamp '{{query_start_date}}'
            {% endif %} 
```

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Review by jeff-dude — APPROVED** [review]
